development.mdx

```diff
@@ -40,10 +40,10 @@ By default, Mintlify uses port 3000. You can customize the port Mintlify runs on
 mintlify dev --port 3333
 ```
 
-If you attempt to run Mintlify on a port that's already in use, you will see an error message:
+If you attempt to run Mintlify on a port that's already in use, it will offer an alternative port:
 
 ```md
```

Comment:
This should probably be `bash` rather than `md` (?)

Reply:
Yeah I find that shell syntax highlighting is good for shell commands but not so much shell output, since it will sometimes highlight keywords like `if` and `for`. Probably the best syntax highlighting here would be plaintext